### PR TITLE
Istio Flakey Test

### PR DESCRIPTION
**Pull request by ChristinaLyu0710-zz**
Contains file that reads past istio-test results of junit.xml located in istio-testing project, sums up the total number of testsuites and cases run and calculates the number of failed tests. The results are written to istio-flakey-test project on Google Cloud Storage.

### CODE REVIEW

**Comment by ChristinaLyu0710-zz** [comment]
I signed it!

**Comment by ChristinaLyu0710-zz** [comment]
I signed it!

File: flakeyTest/jars/a.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (utka) [inline flakeyTest/jars/a.txt]: do you need this file?

**Review by utka — CHANGES_REQUESTED** [review]
Why do we need all this binary files?

File: flakeyTest/jars/a.txt (hunk 1 above)
Comment (ChristinaLyu0710-zz) [reply flakeyTest/jars/a.txt]: I don't need this. Should I delete it? I created this when I created the directory

Comment (utka) [reply flakeyTest/jars/a.txt]: Yes, I suspect you only need a few files with code. the rest should be deleted/

**Review by utka — CHANGES_REQUESTED** [review]
You just need to have 2 files hese Pair and TotalFlakey

File: flakeyTest/jars/a.txt (hunk 1 above)
Comment (ChristinaLyu0710-zz) [reply flakeyTest/jars/a.txt]: Should I also delete all the jar files? But they are needed to compile and run the code

**Comment by icygalz** [comment]
Changes looks to me. Can you also write a README.md file for the folder explaining what your application does and how to use it? We can do it different PR.

/lgtm

**Pull request merged**